src/script/extension/GiphyRepository.ts

```diff
@@ -29,28 +29,22 @@ export interface Gif {
 export interface RandomGifOptions {
   /** How many retries to get the correct size. Default is `3`. */
   maxRetries?: number;
-  /** Maximum gif size in bytes. Default is 3 megabytes. */
+  /** Maximum GIF size in bytes. Default is 3 Megabytes. */
```

Comment:
```suggestion
  /** Maximum GIF size in bytes. */
```

Reply:
Let's remove the numbers for easier maintenance. We the amount of MBs mentioned in `GiphyRepository.CONFIG.MAX_SIZE`.